### DISCUSSION TITLE
Hide mouse/cursor after X seconds

### DISCUSSION THREAD

**Discussion by DoeEensGek** [f1multiviewer/issue-tracker#89] [Feature requests] [answered]
Is it possible to hide mouse after x seconds no movement?
I'm watching 4 streams on 1screen but the mouse stays on top.

Thanks for the great app! It's improving fast.

**Comment by DoeEensGek** [comment]
Maybe this could help.
https://stackoverflow.com/questions/3354239/hiding-the-mouse-cursor-when-idle-using-javascript

**Comment by TheBaconCat** [comment]
If you put your mouse over the main player it won't hide the mouse. But if you instead put your mouse near the bottom where the player controls are. The mouse will hide. This also works in Fullscreen as well!

**Comment by JJWatMyself** [comment]
I think this is a good idea as solves the problem of figuring out where to move the mouse to.

**Reply by TheBaconCat** [reply]
Yeah I agree it would indeed take out the hassle of moving the Mouse to a really specific spot just to hide it. If mouse is inactive in general then hide it makes more sense

**Comment by f1multiviewer** [comment, marked as answer]
This will be fixed in the next release!

**Reply by DoeEensGek** [reply]
Did someone already mention how great you are!! Thanks for the best f1tv application. With the data overlay for onboard, it really changed the way I was watching. Your improving on this project is superb, keep it on. Thanks.

**Comment by DoeEensGek** [comment]
@f1multiviewer broken in latest version. 1.5.8. Worked fine before.

**Reply by f1multiviewer** [reply]
It only works after the window has been active once, it's still working for me, can you check if you activate (focus) the window first, then hold the cursor still, if it works then?
